src/music.cpp

```diff
@@ -0,0 +1,106 @@
+#include "music.h"
+#include "debug.h"
+
+namespace music
+{
+
+std::vector<std::pair<music_id, bool>> music_id_list { std::make_pair( mp3, false ), std::make_pair( instrument, false ), std::make_pair( sound, false ), std::make_pair( title, true ) };
```

Comment:
Consider instead an std::map, it's the data structure that better matches how you use this.

Reply:
Yeah, that sounds like a good suggestion. I applied the suggested change along with other suggestions in this commit.

https://github.com/MegasKomnenos/Cataclysm-DDA/commit/fc934d8a45c15a11f1e08a38baba84709fbecdc2